declare_clippy_lint/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "declare_clippy_lint"
+version = "0.1.65"
```

Comment:
Should be tested to match the top level `Cargo.toml` version. We have a test called ``matches_clippy_version`` or something similar, that already does that for other Clippy crates. Just add this crate there.

Reply:
This is the test I meant and where it should be added: https://github.com/rust-lang/rust-clippy/blob/fe57ab7c6414bbef66097c972e613492b8d95065/tests/versioncheck.rs#L26-L31